tests/IM7-8552.props

```diff
@@ -123,6 +123,15 @@ gs1 = -1.47d0
 gs2 =  5.05d-1
 gs3 = -1.18d-2
 
+// Schaefer Theory (values below are from Schaefer's PPT)
```

Comment:
I do not think we will include Joe's PowerPoint files detailing the derivations and curve fitting in the repository. The off-axis tension and compression tests and data were published in an American Society for Composites paper, see:
https://ntrs.nasa.gov/search.jsp?R=20170010326